src/webgpu/shader/validation/shader_io/align.spec.ts

```diff
@@ -45,6 +48,14 @@ const kTests = {
   },
   const_expr: {
     src: '@align(i_val + 4 - 6)',
+    pass: false,
+    // EXCEPTION: Error: Unexpected validation error occurred:
```

Comment:
note to self:  i_val is a const value of 4.  So this computes to 2, and should be invalid.
LGTM.